VideoLocker/assets/COURSE_NOT_LISTED.htm

```diff
@@ -439,9 +439,8 @@
 
 <p class=MsoNormal style='text-align:justify;text-justify:inter-ideograph'><span
 style='mso-bookmark:OLE_LINK6'><span style='font-size:11.0pt;font-family:Arial;
-mso-bidi-font-family:Arial'>We're working hard to make all of our courses mobile-friendly. 
-If you don't see your course yet, it will be available soon - 
-we're adding more every day!</span></span></p>
+mso-bidi-font-family:Arial'>We're working hard to make all of our courses mobile-friendly. If you can't find a course,
```

Comment:
If you don't mind :smile: 

http://typographyforlawyers.com/straight-and-curly-quotes.html